### ISSUE TITLE
Station Rotation and Propulsion Requests

### ISSUE THREAD

**Issue by Silfryi** [zmaster587/AdvancedRocketry#1355]
Space stations must now all rotate at once,  that makes it so that ships like this hard to construct, and even harder to have solar panels on (Not in the sun half the time)
![latest](https://user-images.githubusercontent.com/40431605/42760662-2334f016-88d9-11e8-99e3-88477950f2e1.png)
This ship would have the ring and main shaft rotate, however the solar panels didn't need to. This also means that the solar panels could be always pointed at the sun, ever if you have your station rotating like I do for the semblance of artificial gravity. What would be most useful was a section like a ring to be attached to your space station so that those connected blocks( solar panels) would not rotate with the station and always be pointed up.

For solid-fuel booster rockets, I would like to have a single, variable height[10-64 blocks tall?] multiblock. They could attach at certain points and be made of an outer shell of machine casings with the engine on the bottom and the fuel inside. They could then drop off after each launch, and have a chance of being recovered w/in 1000 blocks of the launch site.

**Comment by Silfryi** [comment]
Also on the subject of rockets, I saw that there was discussion of solid-fuel booster rockets. For those, I would like to have a single, variable height[10-64 blocks tall] multiblock. They could attach at certain points and be made of an outer shell of machine casings with the engine on the bottom and the fuel inside. They could then drop off after each launch, and have a chance of being recovered w/in 1000 blocks of the launch site.

I like the realism of the fuel, however having the combustion that is supposed to happen IN the rocket happen before to make the fuel makes less sense. I would have the fuel be oxygen and another fuel (Default Hydrogen), that are stored in two separate tanks. Then people could add to the fuel list but still require oxygen.

Lastly, the width max for the launch pad seems to be a bit low. I have managed to build rockets that have barely fit on a pad. I would suggest upgrading it to 32, that leaves people more space to build.
